third_party/VENDOR-LICENSE/github.com/hashicorp/golang-lru/simplelru/lru.go

```diff
@@ -0,0 +1,177 @@
+package simplelru
```

Comment:
same ... I don't think the mechanism of picking up those licenses is bullet proof